storage_service/locations/models/dataverse.py

```diff
@@ -112,63 +212,113 @@ def move_to_storage_service(self, src_path, dest_path, dest_space):
+                    "Invalid value for src_path: %(value)s. Must be a "
+                    "numberic entity_id"
+                )
+                % {"value": src_path}
+            )
         # Fetch dataset info
-        url = 'https://' + self.host + '/api/datasets/' + src_path
-        params = {
-            'key': self.api_key,
-        }
-        LOGGER.debug('URL: %s, params: %s', url, params)
+        url = "https://{}/api/datasets/{}".format(self.host, src_path)
+        params = {"key": self.api_key}
+        LOGGER.debug("URL: %s, params: %s", url, params)
         response = requests.get(url, params=params)
-        LOGGER.debug('Response: %s', response)
+        LOGGER.debug("Response: %s", response)
         if response.status_code != 200:
-            LOGGER.warning('%s: Response: %s', response, response.text)
+            LOGGER.warning("%s: Response: %s", response, response.text)
             raise StorageException(
-                _('Unable to fetch dataset %(path)s from %(url)s') %
-                {'path': src_path, 'url': url})
+                _("Unable to fetch dataset %(path)s from %(url)s")
+                % {"path": src_path, "url": url}
+            )
         try:
-            dataset = response.json()['data']
+            dataset = response.json()["data"]
         except json.JSONDecodeError:
-            LOGGER.error('Could not parse JSON from response to %s', url)
-            raise StorageException('Unable parse JSON from response to %s' % url,)
+            LOGGER.error("Could not parse JSON from response to %s", url)
+            raise StorageException(
+                "Unable parse JSON from response to %s" % url
+            )
 
         # Create directories
         self.space.create_local_directory(dest_path)
 
         # Write out dataset info as dataset.json to the metadata directory
-        os.makedirs(os.path.join(dest_path, 'metadata'))
-        datasetjson_path = os.path.join(dest_path, 'metadata', 'dataset.json')
-        with open(datasetjson_path, 'w') as f:
-            json.dump(dataset, f)
+        os.makedirs(os.path.join(dest_path, "metadata"))
+        datasetjson_path = os.path.join(dest_path, "metadata", "dataset.json")
+        with open(datasetjson_path, "w") as f:
+            json.dump(
+                dataset, f, sort_keys=True, indent=4, separators=(',', ': '))
 
         # Fetch all files in dataset.json
-        for file_entry in dataset['latestVersion']['files']:
-            entry_id = str(file_entry['dataFile']['id'])
-            if not file_entry['label'].endswith('.tab'):
-                download_path = os.path.join(dest_path, file_entry['dataFile']['filename'])
-                url = 'https://' + self.host + '/api/access/datafile/' + entry_id
+        for file_entry in dataset["latestVersion"]["files"]:
+            zipped_bundle = False
+            entry_id = str(file_entry["dataFile"]["id"])
+            if not file_entry["label"].endswith(".tab"):
```

Comment:
You didn't introduce this, but I always find the `if not COND PATH1 else PATH2` construction confusing.
Can we rearrange this to `if COND PATH2 else PATH1`?

Reply:
:+1: